src/unix/process.c

```diff
@@ -197,10 +200,18 @@ static int uv__process_init_stdio(uv_stdio_container_t* container, int fds[2]) {
 
   case UV_CREATE_PIPE:
     assert(container->data.stream != NULL);
+
+    for (j = 0; j < i; j++) {
+      if (options->stdio[j].flags & UV_CREATE_PIPE && options->stdio[j].data.stream == container->data.stream) {
```

Comment:
Long line and can you wrap `options->stdio[j].flags & UV_CREATE_PIPE`in parentheses, otherwise some compilers will complain.

Reply:
:heavy_check_mark: Done.